custom_event_checks/opslevel/service_infrastructure_check/aws-postgres-extended-support-check.yml

```
@@ -0,0 +1,34 @@
+    Group for aurora-postgresql cluster catalog-service-qa"},"dbi_resource_id":"db-G74VR43FATYYWNRXFABLQUQMYY","master_username":"root","associated_roles":[],"db_instance_port":0,"allocated_storage":1,"availability_zone":"us-east-1a","db_instance_class":"db.t4g.medium","storage_encrypted":true,"db_instance_status":"available","db_security_groups":[],"domain_memberships":[],"storage_throughput":0,"certificate_details":{"valid_till":"2024-08-22T17:08:50.000000Z","ca_identifier":"rds-ca-2019"},"db_parameter_groups":[{"parameter_apply_status":"in-sync","db_parameter_group_name":"default.aurora-postgresql11"}],"deletion_protection":false,"monitoring_interval":10,"monitoring_role_arn":"arn:aws:iam::987654321234:role/rds-enhanced-monitoring-catalog-service-qa","publicly_accessible":false,"vpc_security_groups":[{"status":"active","vpc_security_group_id":"sg-041077de84466ebd7"}],"dedicated_log_volume":false,"instance_create_time":"2023-12-04T20:09:22.687999Z","copy_tags_to_snapshot":false,"db_cluster_identifier":"catalog-service-qa","db_instance_identifier":"catalog-service-qa-1","backup_retention_period":7,"pending_modified_values":{},"preferred_backup_window":"04:00-06:00","engine_lifecycle_support":"open-source-rds-extended-support","option_group_memberships":[{"status":"in-sync","option_group_name":"default:aurora-postgresql-11"}],"ca_certificate_identifier":"rds-ca-2019","customer_owned_ip_enabled":false,"auto_minor_version_upgrade":true,"performance_insights_enabled":false,"preferred_maintenance_window":"mon:10:00-mon:11:00","enabled_cloudwatch_logs_exports":["postgresql"],"performance_insights_kms_key_id":"arn:aws:kms:us-east-1:987654321234:key/bda7cdc0-8f2f-435a-a5cc-4bb6e8cbe508","enhanced_monitoring_resource_arn":"arn:aws:logs:us-east-1:987654321234:log-group:RDSOSMetrics:log-stream:db-G74VR43FATYYWNRXFABLQUQMYY","iam_database_authentication_enabled":true,"read_replica_db_instance_identifiers":[]},"__typename":"InfrastructureResource"}]}}]'
+  sampleQueryParams: alias[]=service1&alias[]=service2
+  notes: Postgres 11 has entered into extended support.  This means every database
```

```suggestion
  notes: |
    Postgres 11 has entered into extended support.  This means every database
```
`|` ensures lines under `notes:` are kept together as expected